cli/src/maintenance.rs

```diff
@@ -692,6 +717,32 @@ impl SolidoState {
         None
     }
 
+    /// Try to sell the extra stSOL rewards for UST tokens.
+    pub fn try_sell_anker_rewards(&self) -> Option<(Instruction, MaintenanceOutput)> {
+        let anker_state = self.anker_state.as_ref()?;
+        let reserve_st_sol = anker_state.st_sol_reserve_balance;
+        let st_sol_amount = self
+            .solido
+            .exchange_rate
+            .exchange_sol(Lamports(anker_state.b_sol_total_supply_amount.0))
+            .unwrap();
+        if let Ok(rewards) = reserve_st_sol - st_sol_amount {
```

Comment:
A nicer way to write
```rust
if let Ok(x) = f() {
    Some(g(x))
} else {
    None
}
```
is
```
f().ok().map(g)
```
though if `g` is an ugly closure, in this case because the function returns `Option`, we can also write
```rust
let x = f().ok()?;
g(x)
```
So concretely, we can write
```rust
let rewards = (reserve_st_sol - st_sol_amount).ok()?;
```
and dedent the remainder.

Reply:
Done like it... We still need to check if the rewards is > 0, because it could be 0, but we should return None if it is so.